### PR TITLE
feat: widen electron support

### PR DESCRIPTION
**Pull request by setchy**
Closes https://github.com/max-mapper/menubar/issues/470

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,63 +1,64 @@
...
+		"test": "jest"
+	},
+	"types": "lib/index.d.ts",
+	"dependencies": {
+		"electron-positioner": "^4.1.0"
+	},
+	"devDependencies": {
+		"@amaurym/config": "^1.3.5",
+		"@types/jest": "^25.2.3",
+		"electron": "^29.1.6",
```
Comment (amaury1093) [inline package.json:50]: could you update the package.json inside `examples/` too?

**Review by amaury1093 — COMMENTED** [review]
thanks, will merge after the comment below

**Comment by mraheelkhan** [comment]
@amaury1093 , it is merged but still not available on `npmjs` and it still throws this error
```
npm ERR! Found: electron@29.3.2
npm ERR! node_modules/electron
npm ERR!   dev electron@"^29.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer electron@"^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0 || ^19.0.0 || ^20.0.0 || ^21.0.0 || ^22.0.0 ||  ^23.0.0 || ^24.0.0 || ^25.0.0 || ^26.0.0 || ^27.0.0" from menubar@9.4.0
```

**Comment by setchy** [comment]
> @amaury1093 , it is merged but still not available on `npmjs` and it still throws this error
> 
> ```
> npm ERR! Found: electron@29.3.2
> npm ERR! node_modules/electron
> npm ERR!   dev electron@"^29.0.0" from the root project
> npm ERR!
> npm ERR! Could not resolve dependency:
> npm ERR! peer electron@"^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0 || ^19.0.0 || ^20.0.0 || ^21.0.0 || ^22.0.0 ||  ^23.0.0 || ^24.0.0 || ^25.0.0 || ^26.0.0 || ^27.0.0" from menubar@9.4.0
> ```

i don't believe a new version was published...

**Comment by amaury1093** [comment]
v9.5.0 has been published, with support up to 31 (thanks to #476)

**Pull request merged**